Sources/Swift/Integrations/UserFeedback/SentryFeedbackAPI.swift

```diff
@@ -10,15 +10,17 @@ public final class SentryFeedbackAPI: NSObject {
     /// - seealso: See `SentryOptions.configureUserFeedback` to configure the widget.
     @available(iOSApplicationExtension, unavailable)
     @objc public func showWidget() {
-        SentryFeedbackAPIHelper.showWidget()
+        let feedback = SentrySDKInternal.currentHub().getInstalledIntegration(UserFeedbackIntegration<SentryDependencyContainer>.self) as? UserFeedbackIntegration<SentryDependencyContainer>
```

Comment:
`l`: We could define a method for `SentrySDKInternal.currentHub().getInstalledIntegration(UserFeedbackIntegration<SentryDependencyContainer>.self) as? UserFeedbackIntegration<SentryDependencyContainer>` as it's pretty long code and used in two places.